[2025-07-13T04:05:50.307+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_etl_pipeline.load_processed_covid_data manual__2025-07-13T04:05:38.024105+00:00 [queued]>
[2025-07-13T04:05:50.325+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_etl_pipeline.load_processed_covid_data manual__2025-07-13T04:05:38.024105+00:00 [queued]>
[2025-07-13T04:05:50.326+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-07-13T04:05:50.358+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_processed_covid_data> on 2025-07-13 04:05:38.024105+00:00
[2025-07-13T04:05:50.374+0000] {standard_task_runner.py:60} INFO - Started process 449 to run task
[2025-07-13T04:05:50.378+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'main_etl_pipeline', 'load_processed_covid_data', 'manual__2025-07-13T04:05:38.024105+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmpmvuw0mq0']
[2025-07-13T04:05:50.383+0000] {standard_task_runner.py:88} INFO - Job 24: Subtask load_processed_covid_data
[2025-07-13T04:05:50.470+0000] {task_command.py:423} INFO - Running <TaskInstance: main_etl_pipeline.load_processed_covid_data manual__2025-07-13T04:05:38.024105+00:00 [running]> on host 23913323947a
[2025-07-13T04:05:50.664+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_processed_covid_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-13T04:05:38.024105+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-13T04:05:38.024105+00:00'
[2025-07-13T04:05:50.693+0000] {logging_mixin.py:188} INFO - Iniciando carga de datos procesados a MongoDB en covid_db.processed_covid_data...
[2025-07-13T04:05:51.976+0000] {logging_mixin.py:188} INFO - Dato procesado cargado exitosamente en MongoDB en covid_db.processed_covid_data.
[2025-07-13T04:05:52.046+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-13T04:05:52.081+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=main_etl_pipeline, task_id=load_processed_covid_data, execution_date=20250713T040538, start_date=20250713T040550, end_date=20250713T040552
[2025-07-13T04:05:52.258+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-13T04:05:52.295+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
